src/main/java/eu/europa/ted/eforms/noticeeditor/helper/notice/PhysicalModel.java

```diff
@@ -661,7 +671,7 @@ public static XPath setXmlNamespaces(final DocumentTypeInfo docTypeInfo,
     rootElement.setAttribute(XMLNS, namespaceUriRoot);
 
     final Map<String, String> map;
-    if (sdkVersion.compareTo(new SdkVersion("1.6")) >= 0) {
+    if (sdkVersion.compareTo(DocumentTypeInfo.TEDEFO_1743_SINCE_SDK_VERSION) >= 0) {
```

Comment:
These SINCE_SDK_... stuff is bad practice. We should just remove support of older SDK versions as the application evolves. In the settings we have a list of SDK versions supported by the application. Unsupported versions should be removed from that list and all the SINCE_SDK_ stuff needs to disappear from the code.

Reply:
Yes this pre-dated the release of SDK 1.6, but now we could remove that.

Reply:
This breaks some unit tests based on version 1.5 as the code is now trying to sort the XML elements.